### DISCUSSION TITLE
Lack of images in the search engine against the web version / Falta de imagenes en el Buscador contra la version web

### DISCUSSION THREAD

**Discussion by DarylCamacho** [jgraph/drawio-desktop#1325] [Q&A]
He estado usando el software mucho tiempo pero en las ultimas actualizaciones el buscador en version desktop me sale muy diferente a la version web, fatando imagenes. / I have been using the software for a long time but in the last updates the browser in the desktop version looks very different to the web version, missing images. 

Version WEB : https://ibb.co/r5pvZYH
Version Desktop : https://ibb.co/py1fvgc

**Comment by davidjgraph** [comment]
This has always been the case, unless you were using one of the very early desktop versions. Desktop does not call remote and cannot search extended icons.
